vendor.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/klauspost/compress v1.17.2
 	github.com/miekg/dns v1.1.43
 	github.com/mistifyio/go-zfs/v3 v3.0.1
-	github.com/moby/buildkit v0.11.7-0.20230908085316-d3e6c1360f6e // IMPORTANT: when updating, also update the version in builder/builder-next/worker/worker.go
+	github.com/moby/buildkit v0.11.7-0.20240110143849-d08077628de0 // IMPORTANT: when updating, also update the version in builder/builder-next/worker/worker.go
```

Comment:
Updated the BuildKit version so that we can use `ALPINE_VERSION` in CI (GHA uses the same commit to test buildkit); 


- full diff: https://github.com/moby/buildkit/compare/d3e6c1360f6e...d08077628de0cac2f5081e82c55bdcda918d3da7